webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -427,26 +426,46 @@ def update_corr_index(
         response do not change between ensembles. Yet we need one ensemble to
         determine the default values.
         """
-        if not responses:
+        if ensemble_selection_store:
+            ensembles = [
+                selected_ensemble["value"]
+                for selected_ensemble in ensemble_selection_store["selected"]
+            ]
+        else:
+            ensembles = None
+        if not (responses and ensembles):
```

Comment:
If some ensembles and responses were selected before, we are storing the old values. Is this reasoning correct?

Reply:
That is a very good question. At the end of the day, it is up to user preference I suppose - expected behaviour, I feel, could go either way, it gets cleared, or it don't get cleared, I dunno... I chose this behaviour (of remembering) for now. If you know of a preference, or if you think that this is unintuitive / surprising behaviour, I will adjust it!

indeed *right now* the check is too much, we don't need to check for ensembles and responses, no ensembles implies no responses - I thought a little bit about it, and decided to take "defensive programming" approach here, instead of being more technically correct and concise. In case of future changes, where responses might be selected / exist without ensembles being selected, this is not gonna break.

but if you think this is too defensive or the scenario above is too far-fetched, I can adjust it of course : )

Reply:
Both are good, I was trying to check if my understanding about your intentions was correct, and it was! Keep it going :-)

Reply:
great, thanks : )